flake8_quotes/__init__.py

```diff
@@ -210,21 +149,16 @@ def get_quotes_errors(self, file_contents):
             #   "foo"[0] * 3 = " * 3 = """
             #   "foo"[0:3] = "fo
             #   """foo"""[0:3] = """
-            is_docstring = token in docstring_tokens
+            is_docstring = (token in self.docstring_tokens)
```

Comment:
What's the purpose of the parentheses here? It seems inconsistent with the similar logical operators below

```suggestion
            is_docstring = token in self.docstring_tokens
```

Reply:
Sorry, that's just my personal coding style leaking through. When I contribute to other people's code I try to match their style but sometimes habits take over.

FWIW, I tend to wrap pretty much all expressions in parentheses, make it easier to read for me and make it obvious for cases like this when an expression is boolean